lib/internal/internal.go

```diff
@@ -53,13 +53,13 @@ import (
 // BuildACI takes a Docker layer, converts it to ACI and returns its output
 // path and its converted ImageManifest.
 type Docker2ACIBackend interface {
-	GetImageInfo(dockerUrl string) ([]string, *common.ParsedDockerURL, error)
-	BuildACI(layerIDs []string, dockerURL *common.ParsedDockerURL, outputDir string, tmpBaseDir string, compression common.Compression) ([]string, []*schema.ImageManifest, error)
+	GetImageInfo(dockerUrl string) ([]string, string, *common.ParsedDockerURL, error)
```

Comment:
same comment applies here as above. can we document what `[]string, string` are?